### ISSUE TITLE
PIP mode and Background play outside app

### ISSUE THREAD

**Issue by KyleBroflowski** [AllanWang/Frost-for-Facebook#1782]
Audio only background play(if possible) and a picture in picture that works with the androids pip mode would be perfect. I couldn't find this app in PIP mode section of android settings. For the controlls that appear on PIPmode window, the options can be 
-press center to enter fullscreen back in app
-an icon to initiate audio only background play

**Comment by AllanWang** [comment]
It's great that you have a lot of feature ideas, but note that Frost is somewhat in a maintenance phase. A lot of things are being broken over time which takes time to address, so it's less likely that I will spend much time on new features.

For this, actual PIP is possible, but at the time I created Frost, the API was very new and only available for a few devices. With that, I decided to build my own in app one and probably keep it that way so all android versions could use it.
